tests/test_email_service.py

```
@@ -0,0 +1,863 @@
+                "subject": "Test",
+                "body": "Test body",
+                "headers": {
+                    "X-Disallowed-1": "value1",
+                    "X-Disallowed-2": "value2",
+                    "X-Allowed": "value3"
+                },
+                "caller_id": "service-a"
+            }
+        )
+        
+        with pytest.raises(ValueError) as exc_info:
```

Ruff (0.14.10)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)